en/docs/develop/creating-artifacts/creating-a-message-store.md

```diff
@@ -41,8 +41,8 @@ The message store is created in the `src/main/synapse-config/message-stores` fol
 
 To add a message store to the integration sequence, use the [Store Mediator]({{base_path}}/reference/mediators/store-mediator):
 
-1.	Open to the **Design View** of your [mediation sequence]({{base_path}}/reference/synapse-properties/sequence-properties).
-2.	Drag the [Store Mediator]({{base_path}}/reference/mediators/store-mediator) from the **Palette** and drop it to the relevant position in the [mediation sequence]({{base_path}}/reference/synapse-properties/sequence-properties):
+1.	Open to the **Design View** of your [mediation sequence]({{base_path}}/reference/synapse-properties/mediation-sequences).
```

Comment:
shall we move this out of synapse-properties directory?

Reply:
Fixed via https://github.com/wso2/docs-mi/pull/169/commits/ba4c93370bda2d1d872459578227905e76f39e3c.